### DISCUSSION TITLE
Using Latex templates with jupyter book

### DISCUSSION THREAD

**Discussion by MordicusEtCubitus** [executablebooks/meta#1041] [Q&A]
Dears,

I've seen @JupyterCon 2023  in this great talk "MyST Markdown: Using notebooks in scientific publishing workflows "  https://cfp.jupytercon.com/2023/talk/WWZMSG/ that its seems so easy to build pdf using new latex templates like those: https://github.com/myst-templates

But clearly my notes where not clear enough because I do not understand how to setup jupyter book config and install these templates to be able to use them

Could you help ?

Thanks in advance,

Gaël,

**Comment by rowanc1** [comment]
The PDF templates can be accessed through using the `myst-cli` which is different from JupyterBook ([background here](https://mystmd.org/guide/background#how-do-jupyter-book-and-mystjs-relate)). There is a tutorial which hopefully will get you started [here](https://mystmd.org/guide/quickstart-myst-documents).

You can use the two tools, `jupyterbook` and `myst-cli` on the same content, but the configuration is different currently.

Hopefully that helps!
